pyproject.toml

```diff
@@ -12,6 +12,8 @@ tensorflow-macos = {version = "2.8.0", markers = "platform_machine  == 'arm64'"}
 onnx = "^1.11.0"
 keras-data-format-converter = "0.0.17"
 tensorflow-addons = "^0.17.1"
+transformers = {extras = ["onnx"], version = "^4.20.1"}
+sentencepiece = "^0.1.96"
```

Comment:
Onnx is currently not supported on Mac's M1 chip, In order to support this package on m1 we are putting on onnx dependency the following (you will see it after rebase):
markers = "platform_machine  == 'x86_64'"

Lets check if the two packages you are adding here `transformers` `sentencepiece` can be run on m1 before we add them or put on them the marker to be used only on intel chip